config/forms.php

```diff
@@ -1,5 +1,13 @@
 <?php
-
 return [
 
+    //This stores the forms that this package will generate routes for.
+    'active' => [
```

Comment:
For now, I figured just adding an array that stores a list of active forms, you can reference `routes/api.php ` to see how form routes could then be dynamically generated. This is subject to change.